src/HelloPublisher/Program.cs

```diff
@@ -3,12 +3,18 @@
 using Azure.Messaging.EventGrid;
```

Comment:
Could we rename HelloPublisher and HelloSubscriber to Publisher and Subscriber respectively?